Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -1721,12 +1723,23 @@ DECLARE_SOA_COLUMN(Mass, mass, float);                                  //! mass
 } // namespace oftv0
 } // namespace run2
 
-DECLARE_SOA_TABLE(Run2BCInfos, "AOD", "RUN2BCINFO", run2::EventCuts, //! Legacy information for Run 2 event selection
+DECLARE_SOA_TABLE(Run2BCInfos_000, "AOD", "RUN2BCINFO", run2::EventCuts, //! Legacy information for Run 2 event selection
                   run2::TriggerMaskNext50, run2::L0TriggerInputMask,
                   run2::SPDClustersL0, run2::SPDClustersL1,
                   run2::SPDFiredChipsL0, run2::SPDFiredChipsL1,
                   run2::SPDFiredFastOrL0, run2::SPDFiredFastOrL1,
                   run2::V0TriggerChargeA, run2::V0TriggerChargeC);
+
+DECLARE_SOA_TABLE_VERSIONED(Run2BCInfos_001, "AOD", "RUN2BCINFO", 1,
+                            run2::EventCuts, //! Legacy information for Run 2 event selection
+                            run2::TriggerMaskNext50, run2::L0TriggerInputMask,
+                            run2::SPDClustersL0, run2::SPDClustersL1,
+                            run2::SPDFiredChipsL0, run2::SPDFiredChipsL1,
+                            run2::SPDFiredFastOrL0, run2::SPDFiredFastOrL1,
+                            run2::V0TriggerChargeA, run2::V0TriggerChargeC,
+                            run2::NTPCClusters, run2::NSDDSSDClusters);
+
+using Run2BCInfos = Run2BCInfos_000;
```

Comment:
@ddobrigk Shouldn't be the `001` in place if there are going to be a converter?

Reply:
@victor-gonzalez no, step 1 is to keep using vwrsion 000, then step 2 is to add the converter, and then step 3 is to switch to version 001. This is as intended :-)